fdbsyncd/fdbsync.h

```diff
@@ -10,7 +10,16 @@
 #include "warmRestartAssist.h"
 
 // The timeout value (in seconds) for fdbsyncd reconcilation logic
-#define DEFAULT_FDBSYNC_WARMSTART_TIMER 30
+#define DEFAULT_FDBSYNC_WARMSTART_TIMER 600
+
+// Time to wait to run fdb reconcillation after fdbsyncd replay
+#define FDBSYNC_RECON_TIMER 120
+/*
+ * This is the MAX time in seconds, fdbsyncd will wait after warm-reboot
+ * for the interface entries to be recreated in kernel before attempting to 
+ * write the FDB data to kernel
+ */
+#define WARM_RESTORE_WAIT_TIME_OUT_MAX 180

```

Comment:
Make the name more specific to max time for interface to be restored?

Reply:
ok .. will update